examples/custom_search_method/asha_search_method/utils.py

```diff
@@ -0,0 +1,25 @@
+from typing import Dict
+import random
+
+
+############################################################################
+# User-defined function that generates hyperparameters for each trial.
+# The hyperparameters are passed to a trial in the Create operation.
+# In this example, the model (defined in experiment_files/model_def.py) is expecting
+# the following hyperparameters:
+#   -> global_batch_size,
+#   -> n_filters1
+#   -> n_filters2,
+#   -> learning_rate,
+#   -> dropout,
+#   -> dropout2.
```

Comment:
Should we mention something about how sample_params has to return a specific value rather than a range of values? Because we're not finding the best hyperparameter value in a given set of values in search method. We're doing it only for the values sampled from here. Not sure if that's a clarification that's necessary. But when I think hyperparameter search I think of searching in a range of values.

Reply:
would changing `function that generates hyperparameters for each trial` to `function that generates a set of hyperparameters for each trial` make it more clear that it's about specific values and not about the range?

Reply:
I think `hyperparameters for each trial` is fine. I am using a lengthier phrase: `that generates a combination of hyperparameters (a point in the hyperparameter space) for each trial`.